### ISSUE TITLE
[Bug]: EXPLAIN ANALYZE returns wrong numbers

### ISSUE THREAD

**Issue by aunjgr** [matrixorigin/matrixone#10598]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Recently the `inputRows` and `outputRows` of EXPLAIN ANALYZE returns wrong numbers, and varies every time.

That makes plan/pipeline debugging very difficult.

**Comment by dongdongyang33** [comment]
fixed in #10584

**Comment by aunjgr** [comment]
verified
